README.md

```diff
@@ -55,7 +55,8 @@ if __name__ == "__main__":
 ```
 
 This charm does nothing, though, so you'll typically want to observe some Juju
-events, such as `start`:
+events, such as `start` by calling the `observe()` method, passing it the event
```

Comment:
"such as start by calling"
sounds like you are trying to say something different. Maybe


want to observe some Juju events, such as the `start` event, by calling the `observe()` method, passing the event you want to observe and the handler you want to respond to the event: